core/src/avm2/globals/flash/display/bitmapdata.rs

```diff
@@ -254,18 +254,57 @@ pub fn copy_pixels<'gc>(
                     &src_bitmap_gc_ref
                 };
 
+            let mut alpha_source = None;
+
             if args.len() >= 4 {
+                if let Some(alpha_bitmap) = args
+                    .get(3)
+                    .and_then(|o| o.as_object())
+                    .and_then(|o| o.as_bitmap_data())
+                {
+                    let alpha_point = args
+                        .get(4)
+                        .expect("Missing alpha point!")
```

Comment:
We shouldn't panic because we're missing an argument, it should do whatever flash does - return undefined or something?